src/actions.js

```diff
@@ -124,6 +129,36 @@ export const setUnleashClient = (unleashClient) => ({
   payload: unleashClient,
 });
 
+export const setWalletConnectSessions = (sessions) => ({
+  type: types.SET_WALLET_CONNECT_SESSIONS,
+  payload: sessions,
+});
+
+/**
+ * modal {Object} Modal information to display
+ * modal.show {boolean} Show or hide the modal
+ * modal.type {WalletConnectModalTypes} One of (CONNECT, SIGN_MESSAGE_REQUEST)
+ * modal.onAcceptAction {Object} Action to be dispatched on accept
+ * modal.onRejectAction {Object} Action to be dispatched on reject
+ */
+export const setWalletConnectModal = (modal) => ({
+  type: types.SET_WALLET_CONNECT_MODAL,
+  payload: modal,
+});
+
+export const hideWalletConnectModal = () => ({
+  type: types.SET_WALLET_CONNECT_MODAL,
+  payload: { show: false },
+});
+
+/*
+ * sessionKey {string} The symKey of the connected Session
```

Comment:
WalletConnect uses `symKey` on its documentation for the shared key for the sessions so I think it's best if we keep it as `symKey` to avoid confusion here